doc/rm/register_tool/index.md

```diff
@@ -303,6 +303,11 @@ This signal indicates whether the whole SOC device is in development or producti
 For security peripherals in production mode, it is desired to **not** send an error response, so write misses silently fail, and read misses silently fail, but return either random data (TBD) or all `1`s for response data (i.e. `0xFFFFFFFF` for a 32b register).
 For non-security peripherals, or when in development mode (`devmode_i == 1`) these writes and reads to undefined addresses will return with TL-UL error response.
 
+#### Current Usage of devmode_i
```

Comment:
This sounds good, just two comments:
1) only `devmode_i == 1` returns an error, and I think we set it to 1 everywhere (see line 304 above).
2) we may want to move this paragraph further down, right before `## Register definitions per type` since the following error descriptions `Other error responses (always sent, regardless of ...` seem unrelated to the current usage of devmode.

Reply:
oops, reversed the polarity, let me fix that.